web/src/components/shared/spinner/index.tsx

```diff
@@ -0,0 +1,5 @@
+import './style.css';
+
+export const Spinner = () => {

```

Comment:
Can you please rename this file into ...spinner/Spinner.tsx and export that file from ...shared/index.ts